### ISSUE TITLE
Textredux "hijack" error

### ISSUE THREAD

**Issue by tastyminerals** [rgieseke/textredux#27]
Hi guys, I added `require('textredux').hijack()` into my `~/.textadept/init.lua` file on the first line.
After `reset()` I am receiving the following message:

[](http://imgur.com/a/nOnGE)

**Comment by rgieseke** [comment]
Are you using Textredux master together with the latest Textadept 9 alpha (or nightly build)?
From the upper-case STRIP_TRAILING_SPACES it would seem you are still on Textadept 8 ...

**Comment by tastyminerals** [comment]
Correct, **Textadept 8.7**. Downloade zipped textredux from the web page.

**Comment by rgieseke** [comment]
In that case you can try the commit just before the last one:

https://github.com/rgieseke/textredux/archive/2386c7015bb01cdcf576c1e4e9916226449a67e4.zip

But it should be fine updating to the next alpha version, unless you use any modules that are not updated yet ...
